### ISSUE TITLE
Unicode paper

### ISSUE THREAD

**Issue by ctrlcctrlv** [fhl-net/Lim-Chun-iok_2008_Tai-jip-Tua-su-tian#3]
Hello

Not sure how else to contact you.

I wrote a Unicode paper asking Unicode to encode Taiwanese kana.

You might be interested. You are mentioned.

https://raw.githubusercontent.com/ctrlcctrlv/TaiwaneseKanaUnicodePaper/master/tkana.pdf

**Comment by sih4sing5hong5** [comment]
Thank you for detail proposal.
I forward this github issue to others concern 臺日大辭典.
We can communicate in this issue page.

I have some suggestions,
1. Chinese character `臺羅` in page 7, `Taiwanese government’s offial Tâi-lô ( 臺閩 )` should be  `Taiwanese government’s offcial Tâi-lô ( 臺羅 )`
2. The author of projects in page 9, `Sih Sîng-hông [ 薛丞宏 ]` didn't participate this project,  I just put the dictionary onto github XD. `Lîm Chùn-io̍k [ 林俊育 ]` is the project leader, his email is `limchuniok@gmail.com`. I will forward this issue to him.

Thank you for writing this proposal. It is useful for our literature and document.

**Comment by ctrlcctrlv** [comment]
Thank you! Either of you can reach out to me by email if you prefer as well, copypaste@kittens.ph 

I will make those corrections when I get back at my computer 🙌

**Comment by pcchen** [comment]
@ctrlcctrlv 

Hi, I am Tân, Pektiong [陳柏中] (pcchen). I am quite interested in your proposal and would like to discuss more with you.
What is the best place for the discussion.  Open an issue in https://github.com/ctrlcctrlv/TaiwaneseKanaUnicodePaper ?

**Comment by ctrlcctrlv** [comment]
yes and please feel free to look at my Twitter comment about the paper so you can see what issues have already been raised and which ones I plan to fix

https://twitter.com/HW_BEAT_THAT/status/1295306933935562752?s=19

Thank you for your interest!

**Comment by sih4sing5hong5** [comment]
> Thank you! Either of you can reach out to me by email if you prefer as well, [copypaste@kittens.ph](mailto:copypaste@kittens.ph)
> 
> I will make those corrections when I get back at my computer raised_hands

Thank you for speedy reply. I have forwarded this issue to `Lîm Chùn-io̍k [ 林俊育 ]`by email  and [台日大辭典網路版校對社](https://www.facebook.com/groups/589002898707074/653708865569810/?comment_id=653710588902971&reply_comment_id=653714462235917&notif_id=1597715309507607&notif_t=group_comment&ref=notif)

**Comment by ctrlcctrlv** [comment]
Thank you! Sorry to say guys I do not speak Hokkien but I do speak Japanese if that helps :-)

I'm glad you're all interested in the paper. It is certainly something that has been needed for a long time, I think that the language barrier was the main problem because the Unicode Consortium requires that requests be made in English
